jujugui/static/gui/src/app/assets/javascripts/jujulib/plans.js

```diff
@@ -33,6 +33,34 @@ var module = module;
 
   plans.prototype = {
     /**
+      Get a plan authorization to deploy a charm.
+
+      @public authorizePlan
+      @params env-uuid {String} UUID of the environment the charm is being deployed on
```

Comment:
Please lets call this modelUUID, or just uuid, and document the fact the it's the uuid of a model.